File: src/snippets/If/JavaScript/if.js
const rndNum = () => Math.floor(Math.random() * 5 + 1);

function main() {
	let a = rndNum();
	let b = rndNum();
	let msg;
	console.log(`a = ${a}, b = ${b}`);

	if(a > b) {
		msg = "a > b";
		a++;
	} else if(a < b) {
		msg = "a < b";
		b++;
	} else {
		msg = "a == b";
		a++;
		b++;
	}
	console.log(`${msg}\na = ${a}, b = ${b}`);
}

main()

File: src/snippets/If/C++/if.cpp
#include <iostream>
#include <random>
#include <string>

void print(const int a, const int b) {
	std::cout << "a = " << a << ", b = " << b << '\n';
}
auto rndNum() {
	static std::mt19937 gen(std::random_device{}());
	static std::uniform_int_distribution dis(1, 5);
	return dis(gen);
}

int main() {
	auto a = rndNum();
	auto b = rndNum();
	std::string msg;
	print(a, b);

	if(a > b) {
		msg = "a > b";
		a++;
	} else if(a < b) {
		msg = "a < b";
		b++;
	} else {
		msg = "a == b";
		a++;
		b++;
	}

	std::cout << msg << '\n';
	print(a, b);
	return 0;
}
